notebooks/photo-maker/photo-maker.ipynb

```diff
@@ -67,7 +67,7 @@
   },
```

Comment:
<p>please set lower bound for nncf</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1925/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>done</p>